### ISSUE TITLE
Purchase Functionality of Primary and Secondary DC in Weighbridge Application

### ISSUE THREAD

**Issue by syvasoft** [syvasofterp/org.syvasoft.fortcityminerals#84]
Own Quarry and Subcontract Production Receipt 
1. Just like a Sales Scenario, Primary and Secondary Entry will be made. 
2. Just like a sales DC W/O Info, we need to show separate tab to show Received Qty, TP Qty and Balance TP
3. Secondary DC can be generated based on the primary dc

Purchase from Other Quarry and Crusher
1. TP Number and TP Weight are captured optionally.
2. Only Primary DC
3. No secondary DC
4. Boulder and other aggregates or weight based items will be recorded here.

**Comment by syvasoft** [comment]
Boulder Purchase
1. Own and Subcontract

Finished Products Purchase


Subcontractor and Own Quarry
___________________
Vehicle NO & New Vehicle No
Vehicle Type
Product

Tare
Gross
Net
TP No
TP Weight

Balance Qty
the remaining TP Weight
Product Qty
UOM
Send To
Production Plant
ML - TP Based 
PIT No - Manual entry

TP Is optional for Primary


Finished Material Purchase
1. Manually Enter TP and ML info

Secondary DC should be created manually in Weighbridge app
Weighment should be editable
TP and ML info should be editable

PUrchase Invoice only against Primary DC
Secondary DC TP is used for tracking Pending TP Report

Transit Sales is only primary dc
1. Purchase DC, vendor's DC No is required
2. Create Sales DC should be created based on the third party weighbridge and tag purchase dc if transit sales selected

Capture Vendor Invoice in the Vendor DC then we create the purchase invoice
basically we have Vendor DC#, OUr DC#, Our Invoice No and Vendor Invoice


HIght Priority by 29th
Boulder Purchase from Own Quarry
Purchase quick Entry
Machinery Reference in the Purchase Quick Entry
Cash Book Entry

**Comment by balamuruugan** [comment]
New Fields  in Material Receipt LIne
1. Machinery
2. Issued Qty
3. Rate

While completing Material Receipt, 
1. Create Fuel/Material Issue Entry
2. TF_MInOut


Purchase Quick Entry (Material Receipt + Vendor INvoice) (POS aspect) (MInOut) (Core functionality) event handler
Purchase Order > Material Receipt > Vendor Invoice  (TF_Minout)
